README.md

```diff
@@ -14,31 +14,114 @@ The context stored by this module consists of the following data elements:
 * **refreshToken**: the refresh token used in generating a new access token when one expires
 * **clientId**: the SmartApp's client ID, used in generating a new access token
 * **clientSecret**: the SmartApp's client secret, used in generating a new access token
```

Comment:
These two fields aren't stored anymore, right?

Reply:
Yes these should be removed from what I understand.

Reply:
:thumbsup: